File: Examples/Mvc.VB/Controllers/DocumentViewerController.Protection.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

	    Public Function Protection() As ActionResult
		    Dim documentViewer = New DocumentViewer() With {
			    .Width = 800,
			    .Height = 600,
			    .Resizable = True,
			    .Document = "~/App_Data/ExampleFiles/Default.pdf",
                .DownloadEnabled = False,
                .DownloadAsPdfEnabled = False,
                .PrintEnabled = False,
                .TextSelectionEnabled = False
		    }

		    Return View(documentViewer)
	    End Function

    End Class
End Namespace

File: Examples/Mvc.CS/Controllers/DocumentViewerController.Protection.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.Mvc.CS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult Protection()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 800,
                Height = 600,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.pdf",
                DownloadEnabled = false,
                DownloadAsPdfEnabled = false,
                PrintEnabled = false,
                TextSelectionEnabled = false
            };

            return View(documentViewer);
        }
    }
}